### PR TITLE
chore: cleanup unused dev card eligible setting and associates

### PR DESCRIPTION
**Pull request by rebelchris**
Removed the whole devcard_eligible column from the user table
And everything associated with it, including the email we send to users when they reach this status.

Cleanup amplitude as we no longer support it.

WT-229 #done

### CODE REVIEW

**Review by sshanzel — APPROVED** [review]
It is always so nice to see codebase getting cleaned up. Thank you so much for the effort!

Just one note, do you think we should add a stub value on the returned value from our queries just so it will be compatible with any un-updated consumers?

**Comment by rebelchris** [comment]
@sshanzel I thought about that, but currently 50% of the users would get `0` and the other half `50`.
The default in the featureManagement is also `50` so don't think it would really make much difference.
What do you think?

**Comment by sshanzel** [comment]
Was thinking about the querying part, the un-updated client might receive an error if they asked for the property but are non-existent in the query anymore.

**Comment by sshanzel** [comment]
> Was thinking about the querying part, the un-updated client might receive an error if they asked for the property but are non-existent in the query anymore.

Nevermind this comment. I realized it wasn't actually part of any query.

File: src/entity/User.ts [hunk 1]
```diff
@@ -51,9 +51,6 @@ export class User {
   @Column({ length: 39, nullable: true })
   hashnode?: string;
 
-  @Column({ default: false })
-  devcardEligible: boolean;
```
Comment (idoshamun) [inline src/entity/User.ts:55]: I wouldn't recommend actually dropping this column. I'm not a fan of deleting data 😓

**Review by idoshamun — CHANGES_REQUESTED** [review]
There's nothing like deleting code! 💪

File: src/entity/User.ts (hunk 1 above)
Comment (rebelchris) [reply src/entity/User.ts:55]: No worries, let me put the column back and remove the migration for now :)

**Pull request merged**